docs/js-guide/dependencies.rst

```diff
@@ -112,9 +112,9 @@ Then in your HTML page:
    {% js_entry 'prototype/js/combined_example' %}
 
 The exception to the above is if your page inherits from a legacy page that
-doesn't use a JavaScript bundler, like reports and app manager. This is rare,
+doesn't use a JavaScript bundler, like in app manager. This is rare,
 but one example would be adding a new page to app manager that inherits
-from ``managed_app.html``.
+from ``apps_base.html.html``.
```

Comment:
typo... i think this should be `apps_base.html` rather than `apps_base.html.html`

Reply:
thanks - e9065ef1f6f